roles/openshift_cfme/tasks/main.yml

```diff
@@ -0,0 +1,178 @@
+---
+# PASSED IN ROLE VARIABLES:
+#
+# - template_dir - A remote temp dir to template the PVs in
+#
+# TODO: Refactor creation of this temp dir to a JIT approach to
+# improve idempotency.
+######################################################################
+
+######################################################################
+# Users, projects, and privileges
+
+- name: Ensure the CFME user exists
+  oc_user:
+    state: present
+    username: "{{ openshift_cfme_user }}"
+
+- name: Ensure the CFME namespace exists with CFME user as admin
+  oc_project:
+    state: present
+    name: "{{ openshift_cfme_project }}"
+    display_name: "{{ openshift_cfme_project_description }}"
+    admin: "{{ openshift_cfme_user }}"
+
+- name: Ensure the CFME namespace service account is privileged
+  oc_adm_policy_user:
+    namespace: "{{ openshift_cfme_project }}"
+    user: "{{ openshift_cfme_service_account }}"
+    resource_kind: scc
+    resource_name: privileged
+    state: present
+
+######################################################################
+# Service settings
+
+- name: Ensure bulk image import limit is tuned
+  yedit:
+    src: /etc/origin/master/master-config.yaml
+    key: 'imagePolicyConfig#maxImagesBulkImportedPerRepository'
```

Comment:
I see what you're referencing now, in `inventory/byo/hosts.origin.example` and `inventory/byo/hosts.ose.example` we give this example for tuning the `imagePolicyConfig`:

```
# Configure imagePolicyConfig in the master config
# See: https://godoc.org/github.com/openshift/origin/pkg/cmd/server/api#ImagePolicyConfig
# openshift_master_image_policy_config={"maxImagesBulkImportedPerRepository": 3, "disableScheduledImport": true}
```

Since this is **tech preview** and it is **intended** (it's in the README) to be ran on it's own cluster, I am not going to change this part about setting `imagePolicyConfig#maxImagesBulkImportedPerRepository`. Later on, for the next release, we can find a way to make this work for a wider audience and set of use-cases.